packages/salesforcedx-utils-vscode/src/telemetry/index.ts

```diff
@@ -1,8 +1,17 @@
 /*
- * Copyright (c) 2018, salesforce.com, inc.
+ * Copyright (c) 2020, salesforce.com, inc.
  * All rights reserved.
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-export { TelemetryService, TelemetryBuilder, TelemetryData, Properties, Measurements } from './telemetry';
+/**
```

Comment:
Re-removing the telemetry service index. (the exports were there because of a merge conflict.) See W-8680810 for the follow up work item.